ui/src/components/Sidebar/Sidebar.tsx

```diff
@@ -11,10 +11,15 @@ import {
   signoutIcon,
 } from '../../assets';
 import {SidebarRow} from './SidebarRow';
-
+import {useNavigate} from 'react-router-dom';
 import styles from './styles.module.css';
 
 export const Sidebar: React.FC<{}> = () => {
+  const navigate = useNavigate();
+  const logoutHandler = () => {
+    window.sessionStorage.removeItem('token');
```

Comment:
Logout is a multi step process. Following should be done in order:
1. When user clicks on logout, a `DELETE` request should be sent to the API at `/auth/token:revoke`
2. While the request is being processed, the loading should be shown and the whole screen should be disabled.
3. Once the request succeeds, remove the token from `sessionStorage` and navigate user to `/login`